### ISSUE TITLE
theme paths converted to lowercase

### ISSUE THREAD

**Issue by ael-code** [hacksalot/HackMyResume#172]
In the help message you can find the following two sentences regarding the theme parameter:

```
--theme -t    Path to a FRESH or JSON Resume theme.
```

and under tips section:

```
You can use any FRESH or JSON Resume theme with HackMyResume.
```

The first strange behaviour that I noticed is that it seems that all the path passed through `--theme` parameter are converted to lowercase.

To reproduce the bug you can create a temporary empty directory: `mkdir /tmp/UPPERcase`
and then run `hackmyresume --debug BUILD resume.json --theme="/tmp/UPPERcase"`

First of all the following debug message is printed:
`Verifying /TMP/UPPERCASE theme.`
The question is why it is converted to uppercase here?

Moreover the error message raised is:
`Couldn't find the '/tmp/uppercase' theme. Please specify the name of a preinstalled FRESH theme or the path to a locally installed FRESH or JSON Resume theme.`

And according to `strace`:
`access("/tmp/uppercase", F_OK)          = -1 ENOENT (No such file or directory)`

**Comment by ael-code** [comment]
In the code I found that of the theme parameter is [converted to lowercase](https://github.com/hacksalot/HackMyResume/blob/3cf850ea0e956ca5162c792ad8dc03cc621cf79a/src/verbs/build.coffee#L168) before to be passed to [_verifyTheme](https://github.com/hacksalot/HackMyResume/blob/3cf850ea0e956ca5162c792ad8dc03cc621cf79a/src/verbs/build.coffee#L331)

**Comment by giodamelio** [comment]
I just ran into this problem today. Does anyone know the reasoning behind this?

**Comment by ael-code** [comment]
Actually the fix would be really easy, but I'm waiting for @hacksalot to comment on this

**Comment by hacksalot** [comment]
Thanks for this and your other reports @ael-code. They've been detailed and helpful. Had a minor 18-month delay on this upcoming release but we're moving again now.

IIRC this issue dates back to when we could make stronger assumptions because we were embedding themes and not attempting to support arbitrary JRS themes found in the wild. Will be fixed in 1.9.

**Comment by hacksalot** [comment]
Seems fixed in latest code:

```bash
λ hackmyresume build resume.json to out.all -t ../fresh-standard/fresh-test-themes/fresh/UPPERcase

*** HackMyResume v1.9.0 ***
Reading FRESH resume: resume.json
Applying UPPERCASE theme (3 formats)
Generating TXT  resume: out.txt
Generating JSON resume: out.json
Generating YML  resume: out.yml
```
